composer.json

```diff
@@ -21,7 +21,8 @@
         "guzzlehttp/guzzle": "~6.0",
         "hylianshield/base32-crockford": "^1.0",
         "jms/serializer-bundle": "^3.0",
-        "maennchen/zipstream-php": "^2.1",
+        "laminas/laminas-diactoros": "^2.9",
```

Comment:
Because we already have Guzzle installed, maybe we should use it instead? Thoughts?

- [ ] Keep it, it's probably better.
- [ ] Get rid of it, Guzzle has enough PSR7 coverage for what we need.